infoschema/perfschema/const.go

```diff
@@ -449,17 +451,27 @@ const fieldsInEventsStatementsSummary = " (" +
 	"QUERY_SAMPLE_TEXT LONGTEXT DEFAULT NULL," +
 	"PREV_SAMPLE_TEXT LONGTEXT DEFAULT NULL," +
 	"PLAN_DIGEST VARCHAR(64) DEFAULT NULL," +
-	"PLAN LONGTEXT DEFAULT NULL);"
+	"PLAN LONGTEXT DEFAULT NULL"
 
 // tableEventsStatementsSummaryByDigest contains the column name definitions for table
 // events_statements_summary_by_digest, same as MySQL.
 const tableEventsStatementsSummaryByDigest = "CREATE TABLE if not exists " + tableNameEventsStatementsSummaryByDigest +
-	fieldsInEventsStatementsSummary
+	fieldsInEventsStatementsSummary + ")"
 
 // tableEventsStatementsSummaryByDigestHistory contains the column name definitions for table
 // events_statements_summary_by_digest_history.
 const tableEventsStatementsSummaryByDigestHistory = "CREATE TABLE if not exists " + tableNameEventsStatementsSummaryByDigestHistory +
-	fieldsInEventsStatementsSummary
+	fieldsInEventsStatementsSummary + ")"
+
+// tableClusterEventsStatementsSummaryByDigest contains the column name definitions for table
+// cluster_events_statements_summary_by_digest, same as MySQL.
+const tableClusterEventsStatementsSummaryByDigest = "CREATE TABLE if not exists " + tableNameClusterEventsStatementsSummaryByDigest +
+	fieldsInEventsStatementsSummary + ", ADDRESS VARCHAR(64) DEFAULT NULL)"
```

Comment:
I prefer to move this field to the front.

Reply:
@lonng What do you think？
If change this, need to change another cluster table too, Should better consistent with another cluster table,

Reply:
Prefer to make the `address` as the first column.

Reply:
Addressed, Both cluster table was changed. PTAL @djshow832